core/invoker/src/main/scala/whisk/core/container/package.scala

```diff
@@ -83,7 +83,9 @@ package object container {
         override def toString() = s"$host:$port"
     }
 
-    sealed abstract class ContainerIdentifier(val id: String)
```

Comment:
(e.g. around line 57 in `Container.scala`:

```def details: String = {
        val name = containerName getOrElse "??"```

...the `val` can probably just go, now that it's defined outside (or be replaced with `containerName.map(_.name).getOrElse("??")` if we really prefer that to `anon` in this context)